www/ios/inappbrowser.js

```diff
@@ -60,9 +60,9 @@
             }
         }
 
-        backChannels['preventexitonhide'].subscribe(function(){
+        backChannels.preventexitonhide.subscribe(function(){
```

Comment:
You've missed the point - the original code treats backChannels as an associative array, not an object. Can we change this back.

Reply:
same issue as tests but as we discussed we don't use them